homework/list.js

```diff
@@ -1,44 +1,37 @@
 'use strict';
 
-(function() {
+(function(exports) {
 
+var ListManager = function() {
   var _listNoteContent = [];
-  var _wrapper = document.querySelector('#note-list-wrapper');
+  var _wrapper = null;
+};
 
-  function start() {
-    fetchList(function(data) {
-      updateList(data);
-      drawList();
-      preloadFirstNote();
-    });
-    window.addEventListener('click', function(event) {
-      onNoteOpen(event);
-    });
-  }
+ListManager.prototype = {
 
-  function onNoteOpen(event) {
+  onNoteOpen(event) {
```

Comment:
It's a convention to prefix `on` to an event handler with the corresponding event. So this method should be renamed as `onNoteClicked` in fact.